src/Symfony/Bridge/Monolog/Tests/Formatter/ConsoleFormatterTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Tests\Formatter;
+
+use DateTime;
```

Comment:
we dont use root classes :) should be prefixed with `\`

Reply:
fixed, thank you